.ci/scripts/configure_python.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eEx
+set -o pipefail
+
+# Install conda
+#cd /tmp
+#curl -LO http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+#bash Miniconda3-latest-Linux-x86_64.sh -p /opt/conda -b
+#rm -f Miniconda3-latest-Linux-x86_64.sh
+#export PATH /opt/conda/bin:${PATH}
+
+# Install conda python
+#conda update -y conda
+#conda install -c anaconda -y \
+#    python \
+#    pip \
+#    scikit-learn
+#pip3 install --no-cache-dir python-hostlist
+
+#alternatives --set python /opt/conda/bin/python3
+alternatives --set python /usr/bin/python3
```

Comment:
IMO you need run 1st ``` alternatives --install /usr/bin/python python /usr/bin/python3 1```

Reply:
Could you please elaborate a bit why we need this additional command?
```
# alternatives --install /usr/bin/python python /usr/bin/python3 1
the primary link for python must be /usr/bin/unversioned-python

# update-alternatives --config python
There are 2 programs which provide 'python'.
  Selection    Command
-----------------------------------------------
*  1           /usr/libexec/no-python
 + 2           /usr/bin/python3
Enter to keep the current selection[+], or type selection number: ```